lib/server.js

```diff
@@ -117,15 +120,54 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     routes.notify, brRest.when.prefers.ld,
     validate('continuity-server.notification'),
     asyncHandler(async (req, res) => {
-      // FIXME: we need to authenticate `peerId`; http-signatures should be
-      // added here
+      const verified = await _verifyRequest({req});
+      if(!verified) {
```

Comment:
So in addition to verifying the request, we also need to check whether the `keyId` in the signature matches the expected `peer.id` in the body of the request. Otherwise, we should also send 403. So we'll want to do something like this:

```suggestion
      const {verified, keyId} = await _verifyRequest({req});
      if(!(verified && req.body.peer.id === keyId)) {
```

And make `_verifyRequest` return back more than just the verified status, but also which `keyId` signed.